src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -154,7 +154,9 @@ internal static void ThrowNegateTwosCompOverflow()
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal static unsafe ulong BigMul(uint a, uint b)
         {
-#if TARGET_32BIT
+#if false // TARGET_32BIT
+            // This generates slower code currently than the simple multiplication
+            // https://github.com/dotnet/runtime/issues/11782
```

Comment:
We should probably just remove this (in a separate PR) and ensure the JIT will opportunistically emit `MULX` instead of `MUL` when the hardware supports it instead.